packages/angular-material/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "declarationMap": true,
     "inlineSources": true,
     "sourceMap": true,
+    "target": "es6"
```

Comment:
Why explicitly target es6? Does es2021 not work for you?

Reply:
I fixed the issues and could revert this change